src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example4.txt

```diff
@@ -27,8 +27,8 @@ while (counter < 10)                            // OK
    ++count;
 while (value.incrementValue() < 5); // OK
 switch (num) {
-  case 1: counter++; break;         // OK
-  case 6: counter += 10; break;     // OK
-  default: counter = 100; break;    // OK
+  case 1: counter++; break;         // OK, 'break' is not counted to allow compact view
+  case 6: counter += 10; break;     // OK, 'break' is not counted to allow compact view
+  default: counter = 100; break;    // OK, 'break' is not counted to allow compact view
```

Comment:
This should be changed to two switch blocks, one with violations on both `case N` and `default`, and one good one.

Reply:
Sorry didn't get you can you explain this @nrmancuso